elevation_mapping/include/elevation_mapping/ElevationMapping.hpp

```diff
@@ -175,6 +183,18 @@ class ElevationMapping
    */
   void stopMapUpdateTimer();
 
+  /*!
+   * Get a gridMapMessage of a Submap or rawSubmap
+   * @param position requested position of the submap.
+   * @param length requested length of the submap.
+   * @param layers requested layers of the submap.
+   * @param useRaw if true, returns submap of rawMap, if false, returns submap of fusedMap.
```

Comment:
* @param position the requested position of the submap.
    * @param length the requested length of the submap.
    * @param layers the requested layers of the submap.
    * @param useRaw if true, returns submap of the raw map, if false, returns submap of fused map.